koko.yaml

```diff
@@ -64,8 +64,8 @@ database:
     user: koko
     password: koko
 control_server:
-  tls_cert_path: cluster.crt
-  tls_key_path: cluster.key
+  tls_cert_path: /Users/ggerbino/go/github.com/Kong/koko-private/cp.crt
+  tls_key_path: /Users/ggerbino/go/github.com/Kong/koko-private/cp.key
```

Comment:
Looks like you checked these changes in on accident

Reply:
fml 8)